### ISSUE TITLE
Breaking change ? Unable to use String validate options on array of strings

### ISSUE THREAD

**Issue by neilime** [szokodiakos/typegoose#251]
An error occured after updating typegoose to last version : `Type of phoneNumbers property is not a string.`

Here is the errored property : 
```typescript
  @property({
    lowercase: true,
    required: [true, 'phone_numbers.error.required'],
  })
  public phoneNumbers: string[];
```

I have tried to use `arrayProp` : 

```typescript
  @arrayProp({
    items: String,
    lowercase: true,
    required: [true, 'api.model.agency.contact.phone_numbers.error.required'],
  })
  public phoneNumbers: string[];
```

But option `lowercase` is not defined in `arrayProp` options type.

I think that https://github.com/szokodiakos/typegoose/blob/master/src/prop.ts#L138 should accept array of string too

**Comment by hasezoey** [comment]
i know this is old, but i am working on it currently at https://github.com/Automattic/mongoose/issues/8012

**Comment by hasezoey** [comment]
Update from the mongoose issue:
- it got merged into 5.7
- 5.7 got released
- but there is are still some issues with it https://github.com/Automattic/mongoose/issues/8012#issuecomment-529763148

**Comment by hasezoey** [comment]
@Ben305 this can be closed in favor of https://github.com/typegoose/typegoose/pull/29
